soap/src/java/com/zimbra/soap/JaxbUtil.java

```diff
@@ -1159,7 +1159,9 @@ public final class JaxbUtil {
             com.zimbra.soap.admin.message.CreateS3BucketConfigRequest.class,
             com.zimbra.soap.admin.message.CreateS3BucketConfigResponse.class,
             com.zimbra.soap.admin.message.DeleteS3BucketConfigRequest.class,
-            com.zimbra.soap.admin.message.DeleteS3BucketConfigResponse.class
+            com.zimbra.soap.admin.message.DeleteS3BucketConfigResponse.class,
+            com.zimbra.soap.admin.message.ValidateS3BucketReachableRequest.class,
```

Comment:
there is no handler registered for handling the request, please register handler class for handling the request

Reply:
handler method is there in other repo ( zm-hsm-store )
https://github.com/Zimbra/zm-hsm-store/pull/15